tests/Unit/ControlSystem/Test_TimescaleTuner.cpp

```diff
@@ -189,8 +189,10 @@ void test_increase_or_decrease() {
 
 template <bool AllowDecrease>
 void test_no_change_to_timescale() {
-  const double decrease_timescale_threshold =
-      AllowDecrease ? 1.0e-2 : std::numeric_limits<double>::max();
+  // If we don't allow decrease, then just set the threshold to a super large
+  // value (larger than any test value we'd use). We can't set it to inf or max
+  // otherwise we'll get an FPE.
```

Comment:
Infinity doesn't FPE.  Don't know whether it does what you want, but the test passes with it.

Reply:
It just needs to be a large number so that it will never decrease. Infinity works fine for this so I changed it to that.

Reply:
If you're setting it to infinity, remove the comment that says you can't set it to infinity.

Reply:
Oops!